src/components/screens/ScoreSetCreator.vue

```diff
@@ -95,15 +95,25 @@
                       </div>
                       <div v-else style="position: relative;">
                         <span class="p-float-label">
-                          <Dropdown
+                          <AutoComplete
+                            ref="experimentInput"
                             v-model="experiment"
+                            dropdown
                             :id="$scopedId('input-experiment')"
+                            :suggestions="experimentSuggestionsList"
+                            field="title"
+                            forceSelection
+                            :multiple="false"
                             :options="editableExperiments"
-                            optionLabel="title"
-                            optionValue=""
-                            style="width: 50%"
-                            @change="populateExperimentMetadata"
-                          />
+                            @option-select="populateExperimentMetadata"
+                            @complete="searchExperiments"
+                            @keyup.escape="clearExperimentSearch"
+                            style="width: 100%;"
+                          >
+                            <template #item="slotProps">
+                              {{slotProps.item.urn}}: {{slotProps.item.title}}
+                            </template>
```

Comment:
If we use the `editableExperiments` property directly, it will let us use the `#empty` template to display a nice message when there are no matching experiments instead of just showing a colon. We can add this in addition to the `#item` template.

```
                            <template #empty>
                              <div style="padding: 10px; text-align:center;">
                                No experiments found.
                              </div>
                            </template>
```